src/renderers/dom/shared/ReactDOMInjection.js

```diff
@@ -6,7 +6,7 @@
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  *
- * @providesModule ReactDefaultInjection
+ * @providesModule ReactDOMInjection
```

Comment:
Ah. Thanks. Updated. But I guess I don't technically need it to pass this test. Will revisit when we try to make this test work with Fiber.